compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -74,29 +76,49 @@ pub enum Error {
     #[error("cannot use a dynamic value returned from a runtime-resolved callable")]
     #[diagnostic(code("Qsc.PartialEval.UnexpectedDynamicValue"))]
     #[diagnostic(help("try invoking the desired callable directly"))]
-    UnexpectedDynamicValue(#[label] Span),
+    UnexpectedDynamicValue(#[label] PackageSpan),
 
     #[error("partial evaluation failed with error: {0}")]
     #[diagnostic(code("Qsc.PartialEval.EvaluationFailed"))]
-    EvaluationFailed(String, #[label] Span),
+    EvaluationFailed(String, #[label] PackageSpan),
 
     #[error("unsupported Result literal in output")]
     #[diagnostic(help(
         "Result literals `One` and `Zero` cannot be included in generated QIR output recording."
     ))]
     #[diagnostic(code("Qsc.PartialEval.OutputResultLiteral"))]
-    OutputResultLiteral(#[label] Span),
+    OutputResultLiteral(#[label] PackageSpan),
 
     #[error("an unexpected error occurred related to: {0}")]
     #[diagnostic(code("Qsc.PartialEval.Unexpected"))]
     #[diagnostic(help(
         "this is probably a bug, please consider reporting this as an issue to the development team"
     ))]
-    Unexpected(String, #[label] Span),
+    Unexpected(String, #[label] PackageSpan),
 
-    #[error("failed to evaluate: {0} not yet implemented")]
+    #[error("failed to evaluate: {0} is not supported yet")]
```

Comment:
Nit: I don't like the word "yet" in error messages. Never promise features in an error message :)

Reply:
```suggestion
    #[error("failed to evaluate: {0} is not supported")]
```